src/css/components/_progress.scss

```diff
@@ -101,18 +101,18 @@
 }
 
 .video-js .vjs-load-progress {
-  /* For IE8 we'll lighten the color */
+  // For IE8 we'll lighten the color
   background: ligthen($secondary-background-color, 25%);
-  /* Otherwise we'll rely on stacked opacities */
+  // Otherwise we'll rely on stacked opacities
   background: rgba($secondary-background-color, $secondary-background-transparency);
 }
 
-/* there are child elements of the load progress bar that represent the
-specific time ranges that have been buffered */
+// there are child elements of the load progress bar that represent the
+// specific time ranges that have been buffered
 .video-js .vjs-load-progress div {
-  /* For IE8 we'll lighten the color */
+  // For IE8 we'll lighten the color
```

Comment:
Both of these comments make sense where they are as single line ```//``` comments that will get stripped. Do you really want to move these?